vignettes/poisson.Rmd

```diff
@@ -0,0 +1,359 @@
+is recommended.)
+
+This confirms that goals are relatively rare events in a soccer game with
+each team scoring zero to two goals per match in almost 90 percent of the
+matches. Below we show that this observed frequency distribution can be
+approximated very well by a Poisson distribution which can subsequently
+be used to obtain predicted probabilities for the goals scored in a match.
+
+
+## Basic fitted distribution
+
+In a first step, we simply assume that goals are scored with a constant
+mean over all teams and matches and hence just fit a single Poisson
+distribution for the number of goals. To do so, we obtain a point
+estimate of the Poisson parameter by using the empirical mean
+$\hat \lambda = \bar y = `r round(mean(FIFA2018$goals), digits = 3)`$
+and set up the corresponding distribution object:
+
+```{r}
+p_const <- Poisson(lambda = mean(FIFA2018$goals))
+p_const
+```
+
+In the technical details below we show that this actually corresponds to
+the maximum likelihood estimation for this distribution.
+
+As already illustrated above, the expected probabilities of observing counts of
+$0, 1, \dots, 6$ goals for this Poisson distribution can be extracted using
+the `pdf()` method. A comparison with the observed empirical frequencies yields
+
+```{r}
+expctd <- pdf(p_const, 0:6)
+cbind("observed" = obsrvd, "expected" = expctd)
+```
+
+By and large, all observed and expected frequencies are rather close.
+However, it is not reasonable that all teams score goals with the same probabilities,
+which would imply that winning or losing could just be attributed to "luck" or
+"random variation" alone. Therefore, while a certain level of randomness will
+certainly remain, we should also consider that there are stronger and weaker
+teams in the tournament.
+
+
+## Poisson regression and probabilistic forecasting
+
+To account for different expected performances from the teams in the 2018 FIFA World Cup,
+the `FIFA2018` data provides an estimated `logability` for each team. These
+have been estimated by @Zeileis+Leitner+Hornik:2018 prior to the start of the
+tournament (2018-05-20) based on quoted odds from 26 online bookmakers using
+the bookmaker consensus model of @Leitner+Zeileis+Hornik:2010. The `difference` in
+`logability` between a team and its opponent is a useful predictor for the
+number of `goals` scored.
+
+Consequently, we fit a generalized linear model (GLM) to the data that links
+the expected number of goals per team/match $\lambda_i$ to the linear predictor
+$x_i^\top \beta$ with regressor vector $x_i^\top = (1, \mathtt{difference}_i)$
+and corresponding coefficient vector $\beta$ using a log-link:
+$\log(\lambda_i) = x_i^\top \beta$. The maximum likelihood estimator $\hat \beta$
+with corresponding inference, predictions, residuals, etc. can be obtained in R
+using the `glm()` function with `family = poisson`:
+
+```{r}
+m <- glm(goals ~ difference, data = FIFA2018, family = poisson)
```

Comment:
Perhaps mention we need `glm` and that `glm` comes from `stats`, not `distributions3`?

Reply:
I just point out that it is from "base R" which is hopefully sufficiently clear without creating confusion about what the `stats` package actually is.